src/CLU/Microsoft.Azure.Commands.Compute/VirtualMachine/Config/AddAzureVMAdditionalUnattendContentCommand.cs

```diff
@@ -30,7 +30,7 @@ namespace Microsoft.Azure.Commands.Compute
         ProfileNouns.AdditionalUnattendContent),
     OutputType(
         typeof(PSVirtualMachine))]
-    [CliCommandAlias("vm additional unattend content add")]
+    [CliCommandAlias("vm config additionalunattendcontent add")]
```

Comment:
The idea is that all the cmdlets that change local configuration rather then change the vm in the cloud are grouped under the 'config' heading.

Aliases should not be in conflict (we will add static checks for this)